templates/download/amd/index.html

```diff
@@ -2,19 +2,25 @@
 
 {% block title %}Install Ubuntu on AMD{% endblock %}
 
-{% block meta_description %}Use Ubuntu on AMD for the familiar developer experience and an accelerated path to production.{% endblock meta_description %}
+{% block meta_description %}
+  Use Ubuntu on AMD for the familiar developer experience and an accelerated path to production.
+{% endblock meta_description %}
 
-{% block meta_image %}https://assets.ubuntu.com/v1/2fe9d465-AMD%20download%20page%20meta%20image.png{% endblock meta_image %}
+{% block meta_image %}
+  https://assets.ubuntu.com/v1/2fe9d465-AMD%20download%20page%20meta%20image.png
+{% endblock meta_image %}
 
-{% block meta_copydoc%} https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4 {% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4
+{% endblock meta_copydoc %}
 
 {% block content %}
 
-<section class="p-strip is-deep">
-  <div class="row">
-    <div class="col-3 col-medium-2">
-      <div class="p-strip is-shallow" style="padding-top: 1rem;">
-        {{
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <div class="p-strip is-shallow" style="padding-top: 1rem;">
+          {{
```

Comment:
The linter does not pick this up but could you please update the formatting of the image tags to:
```
{{ image(url="...",
         alt="...",
   ... 
}}
```

Thanks!